new-sdks/user/src/security/CryptoBrowser.js

```diff
@@ -0,0 +1,248 @@
+import {base64Url} from '@token-io/core';
+import Util from '../Util';
+import stringify from 'fast-json-stable-stringify';
+
+const crypto = window.crypto;
+
+// supported algorithms
+const ECDSA = 'ECDSA_SHA256';
+const RSA = 'RS256';
+
+// default to ECDSA and fallback to RSA
+let algorithm = Util.isFirefox || Util.isEdge()
```

Comment:
`Util.isFirefox()` should be a function call (I wonder if this has been forcing chrome and safari to use RSA instead of ECDSA)

Reply:
holy shit i was wondering that too nice catch!